XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs

```diff
@@ -95,88 +95,158 @@ internal static OVRControllerBase[] Controllers
         internal static Controller activeControllerType = Controller.None;
         internal static Controller connectedControllerTypes = Controller.None;
 
+        public static bool Initialized
+        {
+            get
+            {
+                return ovrp_GetInitialized() == Bool.True;
+            }
+        }
+
+        public static float EyeDepth
+        {
+            get
+            {
+                if (!Initialized)
+                    return 0.0f;
```

Comment:
missing braces

Reply:
Oculus API code, can't change to retain license

Reply:
When why is this code checked into the project?
Shouldn't we just reference a submodule?